knative-operator/deploy/resources/knativekafka/controller/eventing-kafka-controller.yaml

```diff
@@ -1153,22 +1153,15 @@ metadata:
   labels:
     app.kubernetes.io/version: v1.14
     knative.dev/crd-install: "true"
-  name: consumergroups.internal.kafka.eventing.knative.dev
+  name: consumers.internal.kafka.eventing.knative.dev
```

Comment:
Hmm, why are the changes here? It's unrelated to UI.
Maybe you need to rebase?

Reply:
I did a rebase, already.

Yes, those changes aren't expected. I'll try rebasing one more.

Reply:
This needs to be merged: https://github.com/openshift-knative/serverless-operator/pull/2813/files

Reply:
Done in 203fc8d231982c3c2bf194c3cecf589d1b1c082e